src/main/kotlin/net/maizegenetics/phgv2/cli/SetupEnvironment.kt

```diff
@@ -0,0 +1,62 @@
+package net.maizegenetics.phgv2.cli
+
+import com.github.ajalt.clikt.core.CliktCommand
+import com.github.ajalt.clikt.parameters.options.option
+import com.github.ajalt.clikt.parameters.options.required
+import java.io.File
+import java.util.stream.Collectors
+
+/**
+ * Class to create the user conda environment for PHG.
+ * The environment will not be activated until a command is run
+ * that requires specific packages.
+ * The purpose of this script is to ensure the environment is created
+ * and available.
+ *
+ * Check if this environment already exists.  If an error occurs when
+ * creating it, do we assume it exists?
+ */
+class SetupEnvironment : CliktCommand() {
+    // The output directory will store the log and error files from the ProcessBUilder conda create command
+    val outputDir by option("-o", "--outputDir", help = "Directory where ProcessBuilder, conda create env command log files will be written")
+        .required()
+
+    // This function uses ProcssBuilder to setup the phgv2 conda environment
+    fun createEnvironment( envFile: String, outputDir:String) {
+        // call ProcessBuilder to execute the conda create env command
+        val builder = ProcessBuilder("conda", "env", "create", "--file", envFile)
+        var redirectOutput = outputDir + "/condaCreate_output.log"
+        var redirectError = outputDir + "/condaCreate_error.log"
+        builder.redirectOutput( File(redirectOutput))
+        builder.redirectError( File(redirectError))
+        println(" begin conda create Command:" + builder.command().stream().collect(Collectors.joining(" ")));
```

Comment:
Should there be a logger here instead of a println?  Also consider not using a java stream to do the joinToString.  It can be done fully in kotlin.

`builder.command().joinToString(" ")`

Reply:
I wasn't sure how to do the logger in clikt.  DO you or Terry know?  I didn't see an example with Brandon's stuff.

I'll fix the joinToString